twisted/plugins/graphite_blueflood_plugin.py

```diff
@@ -137,8 +138,15 @@ def makeService(self, options):
                     value = int(value)
                 options[key] = value
 
+        if options['protocol'] == "MetricPickleReceiver":
+            protocol_cls = MetricPickleReceiver
+        elif options['protocol'] == "MetricLineReceiver":
+            protocol_cls = MetricLineReceiver
```

Comment:
Oh nevermind, it's part of protocol.py where MetricPickleReceiver is also defined.